Converter/include/sampler_poisson.h

```diff
@@ -260,19 +260,18 @@ struct SamplerPoisson : public Sampler {
 					if (isAccepted) {
 						accepted->write(child->points->data_u8 + pointOffset, attributes.bytes);
 						// rejected->write(child->points->data_u8 + pointOffset, attributes.bytes);
-					} else {
+					} else if(numRejected > 0){

```

Comment:
This if is unnecessary. if numRejected is zero then isAccepted will always be true.
Better was the way it was before.

Reply:
There is a slight nuance here. `numRejected > 0` isn't true if it's set to 0. It's only true if it's greater than 0, so it's not guaranteed to run. Though I haven't followed the logic to see if the block makes sense here. Just commenting on the conditional check itself.